test/crio-wipe.bats

```diff
@@ -48,20 +48,26 @@ function test_crio_wiped_images() {
 	# check that the pause image was removed, as we removed a pod
 	# that used it
 	output=$(crictl images)
-	[[ ! "$output" == *"pause"* ]]
+	[[ ! "$output" == *"$IMAGE_USED"* ]]
 }
 
 function test_crio_did_not_wipe_images() {
 	# check that the pause image was not removed
 	output=$(crictl images)
-	[[ "$output" == *"pause"* ]]
+	[[ "$output" == *"$IMAGE_USED"* ]]
 }
 
 function start_crio_with_stopped_pod() {
 	start_crio
 
+	# it must be everything before the tag, because crictl output won't match (the columns for image and tag are separated by space)
+	export IMAGE_USED
```

Comment:
I doubt that `export` is needed here. Is this just to eliminate a shellcheck warning?

Reply:
I think export is needed because this field is used outside this function

Reply:
hm not needed :)